core-connector-template/src/api-spec/core-connector-api-spec-dfsp.yml

```diff
@@ -212,19 +213,24 @@ components:
         homeTransactionId:
           type: string
           example: "HTX123456789"
-        merchantPaymentId:
+        amountType: 
+          type: string
+          example: "RECEIVE"
+          enum:
```

Comment:
Then ENUM should have both SEND and RECEIVE isn't it? if its just one value, using ENUM doesn't seem like a good fit.